src/ImageSharp.Web/Commands/QueryCollectionRequestParser.cs

```diff
@@ -15,18 +14,20 @@ namespace SixLabors.ImageSharp.Web.Commands
     public sealed class QueryCollectionRequestParser : IRequestParser
     {
         /// <inheritdoc/>
-        public IDictionary<string, string> ParseRequestCommands(HttpContext context)
+        public CommandCollection ParseRequestCommands(HttpContext context)
         {
             if (context.Request.Query.Count == 0)
             {
-                return new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+                return new();
```

Comment:
and a static here

Reply:
Same as `PresetOnlyQueryCollectionRequestParser` we need to be able to augment the collection.

Reply:
Yeah you're totally right, and I knew that a year ago when I last worked on this bit of code.

If you feel like popping a comment in to remind me next year, go for it!